docs/Scenario_editor.md

```diff
@@ -2,9 +2,21 @@
 
 ## Getting Started
 
-1. Download Visual Studio from [here](https://visualstudio.microsoft.com/)
+1. Install Unreal Engine **4.23** from [here](https://www.unrealengine.com/en-US/).
 
-    - You'll want to make sure that you have the following options enable, if you do not have Unreal Engine **4.23** already installed there is an option to install it here.
+    - In Epic Games Launcher, go to Marketplace >> Browse. Search and download VehicleSim Dynamics, this is a free plugin. From Epic Games website, you may sign-in and download from [here](https://www.unrealengine.com/marketplace/en-US/product/carsim-vehicle-dynamics).
```

Comment:
This is already mentioned in the LabView documentation (see above). Can we just link to this from there and not have to reiterate the same information? If not, see my suggested edits above on this verbiage.